Dockerfile

```diff
@@ -1,15 +1,36 @@
-# FROM golang:onbuild
-FROM ubuntu:16.04
-ARG DEBIAN_FRONTEND=noninteractive
-RUN apt-get update \
-    && apt-get upgrade -y \
-    && apt-get install -y
+FROM ubuntu:18.04 AS build
+RUN apt-get update && \
+    apt-get -y upgrade && \
+    apt-get install -y wget
+RUN cd /tmp && \
+    wget https://dl.google.com/go/go1.12.7.linux-amd64.tar.gz && \
```

Comment:
How about we just go straight to 1.14? Latest version is 1.14.2. It includes memory optimizations which might benefit Prebid Cache.